src/functions.ts

```diff
@@ -0,0 +1,696 @@
+type constantArr<T> = Readonly<T[]>;
```

Comment:
Type alias and interfaces should start with Uppercase letter.

Reply:
added the upper case letters in the starting of type alias.